app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/govuk_wrapper.scala.html

```diff
@@ -140,6 +141,9 @@
 
 @mainContentHeader = {
     @contentHeader.fold(Html(""))(header => uiMainContentHeader(contentHeader = header))
+    <div class="centered-content">
```

Comment:
if the feature isn't enabled it'd be cleaner not to have an empty `div` here

Reply:
that's removed - not wanted anyway - not sure why HTS use it as it offsets the right alignment with unwanted padding